data-prepper-api/src/main/java/org/opensearch/dataprepper/model/acknowledgements/AcknowledgementSet.java

```diff
@@ -23,13 +22,13 @@
 public interface AcknowledgementSet {
 
     /**
-     * Adds an event to the acknowledgement set. Assigns initial reference
+     * Adds an eventHandle to the acknowledgement set. Assigns initial reference
      * count of 1.
      *
-     * @param event event to be added
+     * @param eventHandle eventHandle to be added
      * @since 2.2
      */
-    public void add(Event event);
+    public void add(EventHandle eventHandle);
```

Comment:
Why not support both for now?

Reply:
Modified to support both using a default method in the interface.